interface/main/calendar/find_patient_popup.php

```diff
@@ -193,7 +193,7 @@ function selpid(pid, lname, fname, dob) {
 <?php elseif (count($result) == 0) : ?>
 <div id="searchstatus" class="noResults"><?php echo htmlspecialchars(xl('No records found. Please expand your search criteria.'), ENT_NOQUOTES); ?>
```

Comment:
Brought this script into main codebase (PR is at https://github.com/openemr/openemr/pull/1129)